components/AnimationTiming/examples/AnimationTimingExample.swift

```diff
@@ -0,0 +1,162 @@
+/*
```

Comment:
Nit: Please space-out the copyright a bit here as well. Makes it easier to read. :)